### ISSUE TITLE
Why can't accessibility work normally in Iceraven?

### ISSUE THREAD

**Issue by popyoung** [fork-maintainers/iceraven-browser#614]
I have a personal app to manage my password. It uses ACTION_SET_TEXT to set text to the EditText. In other apps, it works well. But in Iceraven, nothing happens. I just wonder why and is there a solution?

**Comment by seniordevel** [comment]
I am not sure why it does not work.
But in my opinion no other app should be allowed to manipulate a browser.

I think it is for safety and security reasons and should not be changed.

**Comment by popyoung** [comment]
> But in my opinion no other app should be allowed to manipulate a browser.

I agree with that to some extent. But accessibility is designed to do these things. Although I have to admit it is abused on too many occasions.
